packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -597,12 +598,10 @@ export function TransferPanelMain({
 
   // whenever the user changes the `amount` input, it should update the amount in browser query params as well
   useEffect(() => {
-    setQueryParams({ amount })
-
     if (isMaxAmount) {
       setMaxAmount()
     }
-  }, [amount, isMaxAmount, setMaxAmount, setQueryParams])
```

Comment:
there's a case when user types a string on the url, the amount doesn't get reset to an empty string

try `http://localhost:3000/?amount=SD&destinationChain=arbitrum-sepolia&sourceChain=sepolia`

probably should handle it in another hook but that was part of what the deleted lines handled

Reply:
Ideally, we would sanitize that on the first render on the server, but I have reverted this change